documentation/src/main/asciidoc/topics/proc_cli_logging.adoc

```diff
@@ -63,3 +63,36 @@ configuration, which means the root configuration is used instead:
 ----
 logging remove org.infinispan
 ----
+
+== Access logging
+
+The command also allows for enabling access logs during runtime to analyze issues.
+You can enable the access logging for a specific endpoint and utilize the CLI command to parse those files to generate server-side metrics.
+This technique is useful in identifying whether latency is an issue from the connection between client and server or intra-cluster.
+
+The workflow for this analysis involves a few steps:
+
+. Enable access logging with the CLI for the endpoint you are analyzing.
++
+[source,options="nowrap",subs=attributes+]
+----
+logging set --level=TRACE org.infinispan.RESP_ACCESS_LOG
+----
++
+In this example, enabling the access log for the RESP endpoint.
+
+. Run the tests to reproduce the issue or workload to measure.
+. Disable access logging with the CLI.
++
+[source,options="nowrap",subs=attributes+]
+----
+logging set --level=INFO org.infinispan.RESP_ACCESS_LOG
+----
+
+. Download the access log files.
++
+The report command is also capable of downloading the log files.
```

Comment:
```suggestion
The `report` command is also capable of downloading the log files.
```
I don't know what formatting we want to do for the commands, but they should definitely stand out somehow.